### PR TITLE
What's new in DevTools 118

### PR DESCRIPTION
**Pull request by sofiayem**
ditto

### CODE REVIEW

**Review by rachelandrew — APPROVED** [review]

File: site/en/docs/devtools/workspaces/index.md [hunk 1]
```diff
@@ -41,30 +41,30 @@ your computer. For example, suppose:
...
 
 If you're using a modern framework, it probably transforms your source code from a format that's
-easy for you to maintain into a format that's optimized to run as quickly as possible. Workspaces is
+easy for you to maintain into a format that's optimized to run as quickly as possible. Workspace is
 usually able to map the optimized code back to your original source code with the help of [source
 maps][4].
 
 The DevTools community works to support the capabilities provided by source maps across a [variety of frameworks and tools](/docs/devtools/javascript/source-maps/#use_a_supported_preprocessor).
-If you run into issues while using Workspaces with your framework of choice, or you get it working
+If you run into issues while using Workspace with your framework of choice, or you get it working
```
Comment (jecfish) [inline site/en/docs/devtools/workspaces/index.md:55]: should be lowercase?

File: site/en/docs/devtools/workspaces/index.md [hunk 2]
```diff
@@ -229,7 +229,7 @@ to your site. There's a way to have the **Sources** panel open alongside other p
 Congratulations! You now know how to save to sources the changes you make in DevTools to your locally hosted website.
 {% endAside %}
 
-You can set up multiple folders as **Workspaces**. All such folders are listed in {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/9gzXiTYY0nZzBxGI6KrV.svg", alt="Settings.", width="24", height="24" %} [**Settings** > **Workspace**](/docs/devtools/settings/#workspace).
+You can set up multiple folders in a **Workspace**. All such folders are listed in {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/9gzXiTYY0nZzBxGI6KrV.svg", alt="Settings.", width="24", height="24" %} [**Settings** > **Workspace**](/docs/devtools/settings/#workspace).
```
Comment (jecfish) [inline site/en/docs/devtools/workspaces/index.md:232]: capital or lowercase? There's only one workspace in DevTools. should it be the workspace?

File: site/en/docs/devtools/workspaces/index.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
 ---
 layout: "layouts/doc-post.njk"
```
Comment (jecfish) [inline site/en/docs/devtools/workspaces/index.md:2]: Not exactly sure should I just review the proposed changes or the entire docs. If it's the docs, there are places where we still refer workspace as "Filesystem". And there are still:
- inconsistent capital / lowercase of workspace
- inconsistent plural and singular workspaces

File: site/en/blog/new-in-devtools-118/index.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/b16ef33f36c694dd67fe07a24b0ff5da98ab9307 #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/bd7b236205de8c3e7cb3e67d83e8c4f1ada44ad4 #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/11ca806035cccb4af046583e8b8d7aa7dc1af0f1 #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/33964c8dfa442b928c5320a8f82d14287ab06a8e #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/c4a2a6833828f4425e5e5add1b7aec6aafb3a451 #}
+
+Chromium issues: [1470401](https://crbug.com/1470401), [1471301](https://crbug.com/1471301), [1474108](https://crbug.com/1474108), [1468631](https://crbug.com/1468631).
+
+## Emulate prefers-reduced-transparency media feature {: #reduced-transparency }
```
Comment (jecfish) [inline site/en/blog/new-in-devtools-118/index.md:161]: This should move up to before Lighthouse 11 given the content priority.

File: site/en/blog/new-in-devtools-118/index.md [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+- Various [**Application** > **Preloading**](/blog/new-in-devtools-117/#preloading) improvements, most notably sortable grids and revised rule set details ([1410709](https://crbug.com/1410709)).
+- Various improvements of the command editor in [**Protocol monitor**](/blog/new-in-devtools-117/#protocol-monitor), most notably warnings on wrong input, editing a sent command, editor for object parameters without predefined keys, support for enums undefined by references, objects without type reference, filter commands by substring matches, and more ([1448050](https://crbug.com/1448050)).
+- **Performance** flame chart gets a border around the total box on the pie chart ([1470147](https://crbug.com/1470147)).
+- **Sources** now doesn't treat dashes as word characters in CSS ([1471354](https://crbug.com/1471354)).
+- Autocomplete now always sorts CSS-wise keywords at the end.
+- RegEx filters now support spaces ([1346936](https://crbug.com/1346936)).
+- **Elements** fixed media query feature detection ([1472693](https://crbug.com/1472693)).
+
+## [Experimental] New Application > Storage > Storage buckets section {: #storage-buckets }
```
Comment (jecfish) [inline site/en/blog/new-in-devtools-118/index.md:188]: added a question in the review docs on this section.

**Pull request merged**